src/parsing.jl

```diff
@@ -11,8 +11,16 @@ function encode_ids!(g)
     end
 end
 
-function parse_graphml(filename::String)::MetaDiGraph
+function parse_graphml(filename::String, duration_scale::Float64)::MetaDiGraph
```

Comment:
```suggestion
function parse_graphml(filename::String, duration_scale::Float64=1.0)::MetaDiGraph
```

The tests are failing because there we are also calling the `parse_graphml` but it wasn't updated to use the `duration_scale` argument. Giving a default value should solve this.